### DISCUSSION TITLE
Is LiteDb dead?

### DISCUSSION THREAD

**Discussion by stigzler** [litedb-org/LiteDB#2402] [General]
Loads of unaddressed bugs, unanswered issues and forum virtually dead. Is this now abandonware? What a shame.

**Comment by devmaestro1** [comment]
Looks like there was a commit only 2 weeks ago?

**Reply by stigzler** [reply]
Happy days! Good to see it's still alive!

**Comment by JKamsker** [comment]
Always welcome to join the discussion at https://discord.gg/u8seFBH9Zu
